docs/sources/reference/commandline/cli.md

```diff
@@ -349,6 +350,31 @@ https://linuxcontainers.org/) via the `lxc` execution driver, however, this is
 not where the primary development of new functionality is taking place.
 Add `-e lxc` to the daemon flags to use the `lxc` execution driver.
 
+### Daemon user namespace support
```

Comment:
The question "why can't I set this per container" is easy to understand.  If we provide an answer, it should be equally easy to understand.

For example, [here the first three sentences](https://www.safaribooksonline.com/library/view/beautiful-code/9780596510046/ch17s03.html) provides a pretty good explanation of filesystem layering.  These sentences have a Grade Level 16 of readability. Most of our documentation is written at 8 or lower.  Effectively, most of our readers would find this very elegantly written explanation hard to parse.

Moreover, in designing the information in this section, do I really want to distract the reader from their actual focus task _only_ to provide three long sentences to explain part of what is meant by this single phrase _Due to the requirement that the filesystem layers need to have user/group ownership modification to make this remapping useful,_?  

If we need to say anything, I'd rather write "Due to file system limitations, you cannot map on a per container basis"  and move on.  Yes, I agree with you, there are  readers at the contributor level who may want the implementationdetail but we have other avenues such as IRC or forums for those questions.